accessibility-checker-engine/src/v4/util/CommonUtil.ts

```diff
@@ -60,7 +60,7 @@ export function getInvalidRoles(ruleContext: Element) {
     
     let invalidRoles = [];
 
-    if (allowedRoles && allowedRoles.includes('any'))
+    if (allowedRoles.includes('any'))
```

Comment:
this is a clean up for an extra check (never happen) because the line 58 already has the check !allowedRoles:
 line 58:     if (!allowedRoles && allowedRoles.length === 0)

Reply:
resolved